### ISSUE TITLE
Tensor转numpy数组速度瓶颈

### ISSUE THREAD

**Issue by TALQinYong** [WenmuZhou/DBNet.pytorch#7]
@WenmuZhou 
老哥
        bitmap = _bitmap.cpu().numpy()  # The first channel
        pred = pred.cpu().detach().numpy()
前向测试的时候，这里会极大的限制速度，这个问题你怎么解决的

**Comment by WenmuZhou** [comment]
这个主要是因为gpu上还有操作没有完成，需要等待完成。 参考这个https://blog.csdn.net/u013548568/article/details/81368019

**Comment by TALQinYong** [comment]
@WenmuZhou 哥，你说的这个问题我下去了解了一下，但还是有一点我想不通，我跑fpn+db的前向测试，没秒大概23fps，远远到不了实验中展示的值，而且我对后处理的每个函数都单独测时间，发现用时最多的就是数据从gpu转到cpu，你实验的时候没有遇到这个问题吗？

**Comment by WenmuZhou** [comment]
没有a

**Comment by Whu-wxy** [comment]
看到原作者测试FPS为了忽略IO时间，把模型推理和后处理部分循环处理很多次取平均，感觉这样测试FPS不太对啊。

我测试过，只有第一次后处理时需要进行数据同步和gpu转移到cpu的速度很慢，大概有100ms，后面所有的后处理因为都不需要这一步，速度很快。原作者测试FPS相当于把第一次的同步和IO平摊到了很多次后处理里面，感觉这样的测试不对啊。
实际应用的时候怎么能不考虑这一段时间呢？！
